ArcadiaParties-SPA/src/app/app.effects.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { TestBed, inject } from '@angular/core/testing';
+import { provideMockActions } from '@ngrx/effects/testing';
```

Comment:
That's great, but we could skip testing for this project. Otherwise you can to not meet deadlines.